### PR TITLE
Add absolute number to number range

### PR DESCRIPTION
**Pull request by ruthvik-17**
bug fixes in number detection which affect the number range detector.

### CODE REVIEW

File: ner_v2/detectors/numeral/number/standard_number_detector.py [hunk 1]
```diff
@@ -151,8 +161,10 @@ def _get_unit_from_text(self, detected_original, processed_text):
 
         # add re.escape to handle decimal cases in detected original
         detected_original = re.escape(detected_original)
-        unit_matches = re.search(r'\W+((' + self.unit_choices + r')[\.\,\s]*' + detected_original + r')|(' +
-                                 detected_original + r'\s*(' + self.unit_choices + r'))\W+', processed_text,
+        unit_matches = re.search(r'\W+((' + self.unit_choices + r')[.,\s]*' + detected_original +
+                                 r')\W+|\W+(' + detected_original + r'\s*('
+                                 + self.unit_choices + r'))\W+',
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number/standard_number_detector.py]: W503 line break before binary operator

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by ruthvik-17** [comment]
retest this please

**Comment by amansrivastava17** [comment]
retest this please

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

File: ner_v2/detectors/numeral/number/standard_number_detector.py [hunk 2]
```diff
@@ -209,11 +221,15 @@ def _detect_number_from_words(self, number_list=None, original_list=None):
         numeral_text_list = re.split(r'[\-\:]', self.processed_text)
         for numeral_text in numeral_text_list:
             numbers, original_texts = get_number_from_number_word(numeral_text, self.numbers_word_map)
-            for number, original_text in zip(numbers, original_texts):
+            full_list = zip(numbers, original_texts)
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number/standard_number_detector.py]: W1637 [34mzip built-in referenced when not iterating[0m

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

File: ner_v2/detectors/numeral/number/standard_number_detector.py [hunk 3]
```diff
@@ -209,11 +221,19 @@ def _detect_number_from_words(self, number_list=None, original_list=None):
         numeral_text_list = re.split(r'[\-\:]', self.processed_text)
         for numeral_text in numeral_text_list:
             numbers, original_texts = get_number_from_number_word(numeral_text, self.numbers_word_map)
-            for number, original_text in zip(numbers, original_texts):
+            full_list = list(zip(numbers, original_texts))
+            """
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number/standard_number_detector.py]: W291 trailing whitespace

File: ner_v2/detectors/numeral/number/standard_number_detector.py [hunk 4]
```diff
@@ -209,11 +221,19 @@ def _detect_number_from_words(self, number_list=None, original_list=None):
         numeral_text_list = re.split(r'[\-\:]', self.processed_text)
         for numeral_text in numeral_text_list:
             numbers, original_texts = get_number_from_number_word(numeral_text, self.numbers_word_map)
-            for number, original_text in zip(numbers, original_texts):
+            full_list = list(zip(numbers, original_texts))
+            """ 
+            list() is added to above zip as in python 3, zip() returns a zip object instead of zip function and
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number/standard_number_detector.py]: W291 trailing whitespace

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**